datasync/downloader/downloader.go

```diff
@@ -1578,6 +1624,14 @@ func (d *Downloader) commitFastSyncData(results []*fetchResult, stateSync *state
 func (d *Downloader) commitPivotBlock(result *fetchResult) error {
 	block := types.NewBlockWithHeader(result.Header).WithBody(result.Transactions)
 	logger.Debug("Committing fast sync pivot as new head", "number", block.Number(), "hash", block.Hash())
+	if result.StakingInfo != nil {
+		if err := reward.AddStakingInfoToDB(result.StakingInfo); err != nil {
+			logger.Error("Inserting downloaded staking info is failed", "err", err)
+			return fmt.Errorf("failed to insert the downloaded staking information: %v", err)
+		} else {
+			logger.Info("Imported new staking information", "number", result.StakingInfo.BlockNum)
```

Comment:
If the string is exactly the same, later it is not easy which statement is executed. Could you little bit differentiate the log string between commitPivotBlock() and commitFastSyncData()?

Reply:
Okay. I have updated it.